src/assets/translations/fr.json

```diff
@@ -34,6 +34,7 @@
     "off": "Désactiver",
     "pause": "Pause",
     "play": "Play",
+    "playback": "Play",
```

Comment:
Playback is Play even in English?
You should explain in the PR description why we need another field which seems to echo play and isn't used in these changes.

Reply:
@robwalch Changed the PR description.